src/gluonts/evaluation/_base.py

```diff
@@ -101,6 +103,11 @@ class Evaluator:
     chunk_size
         Controls the approximate chunk size each workers handles at a time.
         Default is 32.
+    mask_invalid_values
```

Comment:
Now I wonder if `ignore_invalid_values` is an even better name.

Reply:
Sure, im fine with that.